template/pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
 pythonpath = .
-addopts = --disable-warnings -p no:warnings
+addopts = -s --disable-warnings -p no:warnings
```

Comment:
btw, I forgot to mention, I already handle verbosity via a `--verbose` option.

https://github.com/informalsystems/atomkraft/blob/78e6c541f8146a199982ca158573daa87ce205de/atomkraft/test/trace.py#L80-L82

If you think, `--verbose` should be on by default, then I can make a `--silent` (`--noverbose`) flag instead.

Reply:
oh I see! then I will revert this change, it's not needed. I was only used to see the output, so was surprised when it disappeared:)